Pod/Classes/MKMapView+Rx.swift

```diff
@@ -176,8 +176,13 @@ extension MKMapView {
         let source = rx_delegate.observe("mapView:annotationView:didChangeDragState:fromOldState:")
             .map { a in
                 return (view: try castOrThrow(MKAnnotationView.self, a[1]),
-                    newState: try castOrThrow(MKAnnotationViewDragState.self, a[2]),
-                    oldState: try castOrThrow(MKAnnotationViewDragState.self, a[3]))
+                    newState: try castOrThrow(UInt.self, a[2]),
+                    oldState: try castOrThrow(UInt.self, a[3]))
+            }
+            .map { (view, newState, oldState) in
+                return (view: view,
+                    newState: MKAnnotationViewDragState(rawValue: newState)!,
+                    oldState: MKAnnotationViewDragState(rawValue: oldState)!)
```

Comment:
The second `map` operator looks to not so ugly, if I would just cast it the first one

What do you think?